.github/workflows/container-images-cd.yml

```diff
@@ -122,3 +122,21 @@ jobs:
                       {
                         "image_tag": "${{ steps.build.outputs.digest }}"
                       }
+
+            - name: Check for changes that affect temporal worker
+              id: check_changes_temporal_worker
+              run: |
+                  echo "::set-output name=changed::$(git diff --name-only HEAD^ HEAD | grep '^posthog/temporal/|^posthog/batch_exports/|^posthog/management/commands/start_temporal_worker.py$' || true)"
```

Comment:
I think you need to use egrep when checking several patterns:

<img width="423" alt="image" src="https://github.com/PostHog/posthog/assets/115043334/04c9f156-b53e-4dce-be85-1ca29b84aa49">

Reply:
Going with `grep -E` as `egrep` is obsolete, thank you!